### ISSUE TITLE
Can't resolve dependencies on Intellij IDEA 2019.1.1

### ISSUE THREAD

**Issue by exoego** [scala-js/scala-js#3638]
When I import this repository on Intellij IDEA 2019.1.1 using Scala plugin for the IDE, sbt could not open the project due to following error.

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.portable-scala#sbt-platform-deps;1.0.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn] 	Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes.
[warn] 		org.portable-scala:sbt-platform-deps:1.0.0 (scalaVersion=2.12, sbtVersion=0.13)
[warn]
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.portable-scala:sbt-platform-deps:1.0.0 (scalaVersion=2.12, sbtVersion=0.13) ((sbt.BuildExtra) Defaults.scala#L2092)
[warn] 		  +- org.scala-js:sbt-scalajs:0.6.28-SNAPSHOT (scalaVersion=2.12, sbtVersion=0.13)
...
[error] (sbtPlugin/*:ssExtractDependencies) sbt.ResolveException: unresolved dependency: org.portable-scala#sbt-platform-deps;1.0.0: not found
[error] (sbtPlugin/*:update) sbt.ResolveException: unresolved dependency: org.portable-scala#sbt-platform-deps;1.0.0: not found
```

But when I use sbt directly on terminal, I can open the project and even can run `compile`, `test` or `fastOptJS` tasks.
* openjdk version "11.0.2" 2019-01-15
* sbt 1.2.8

Intellij IDEA gives better developer experience (code completion, instant compiler error, jump to definition and so on) compared to using sbt only, so I think supporting IDEA is benefical for contributors.

Any ideas to solve?

**Comment by sjrd** [comment]
There was an attempt in #3511. You might want to try and revive it.

**Comment by exoego** [comment]
@sjrd 
What was a blocker for #3511 ?
Looks like CI passes most jobs and fails only 2 jobs due to 
`java.io.FileNotFoundException: /data/sjsjenkins/.jenkins/jobs/scalajs-main/branches/PR-3511/builds/2/log (No such file or directory)` which seems not related to sbt (not sure)

**Comment by sjrd** [comment]
Fixed in a52acbe468d115bd789358b1e0033785538bc125.
